art/classifiers/tensorflow.py

```diff
@@ -766,6 +774,10 @@ def fit(self, x, y, batch_size=128, nb_epochs=10, **kwargs):
         # Apply preprocessing
         x_preprocessed, y_preprocessed = self._apply_preprocessing(x, y, fit=True)
 
+        # Check label shape
+        if isinstance(self._loss_object, tf.keras.losses.SparseCategoricalCrossentropy):
```

Comment:
If following comment above, we could replace this with `self._reduce_labels`.

Reply:
Done.